apps/web/src/services/asset-inventory/collector/modules/CollectorContents.vue

```diff
@@ -143,14 +161,20 @@ const handleChange = async () => {
         .left-area-wrapper, .search-wrapper, .tool {
             margin-bottom: 0;
         }
+
+        @screen tablet {
+            .left-area-wrapper, .search-wrapper, .tool {
+                margin-bottom: 1rem;
+            }
+        }
     }
 
-    /* custom design-system component - p-data-loader */
-    :deep(.p-data-loader) {
-        .data-wrapper {
+    .collector-list-wrapper {
+        .collector-list {
             @apply grid grid-cols-2 gap-4;
 
-            .p-card {
+            /* custom design-system component - p-card */
+            :deep(.p-card) {
```

Comment:
I changed the 'collector-list-wrapper' to 'div' instead of using 'p-data-loader', but the style of the card was not applied. As a temporary solution, I used 'deep' but added a 'FIXME' comment for it!

Reply:
Oh, that's cool of using 'FIXME' comment for it :) Thank you for replying it!